pkg/reconciler/apiserversource/apiserversource_test.go

```diff
@@ -90,6 +90,8 @@ func init() {
 	_ = appsv1.AddToScheme(scheme.Scheme)
 	_ = corev1.AddToScheme(scheme.Scheme)
 	_ = duckv1alpha1.AddToScheme(scheme.Scheme)
+	// Set path
+	sinkTargetURI.Path = sinkURIReference
```

Comment:
I found complicated figuring out the full URL because it's not set in one place.
What if we use something like:
```golang
sinkTargetURI = func() apis.URL {
    u := apis.HTTP(sinkDNS)
    u.Path = sinkURIReference
    return u
}()
```
or 
```golang
sinkTargetURI = targetURL(sinkDNS, sinkURIReference)

// ...

func targetURL(host, path string) apis.URL {
   u := apis.HTTP(sinkDNS)
   u.Path = sinkURIReference
   return u
}
```